search-commons/src/main/java/no/unit/nva/search/common/QueryFilter.java

```diff
@@ -44,6 +49,6 @@ public void add(QueryBuilder builder) {
     public String toString() {
         return filters.values().stream()
                 .map(QueryBuilder::toString)
-                .collect(Collectors.joining(COMMA));
+                .collect(Collectors.joining(COMMA, "{", "}"));
```

Comment:
This appears to "double-wrap" the values, is this intended?

Example output from this `toString` in a random test:
```
{{
  "terms" : {
    "status.keyword" : [
      "PUBLISHED",
      "PUBLISHED_METADATA"
    ],
    "boost" : 1.0,
    "_name" : "status"
  }
}}
```

Reply:
it is a simple way to resolve output problems in aws-log, replaced with '[' ']'